src/main/java/com/github/alex1304/jdash/client/GDUserProfileCommentRequest.java

```diff
@@ -0,0 +1,78 @@
+package com.github.alex1304.jdash.client;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import com.github.alex1304.jdash.entity.GDComment;
+import com.github.alex1304.jdash.exception.GDClientException;
+import com.github.alex1304.jdash.util.GDPaginator;
+import com.github.alex1304.jdash.util.Indexes;
+import com.github.alex1304.jdash.util.ParseUtils;
+import com.github.alex1304.jdash.util.Routes;
+import com.github.alex1304.jdash.util.Utils;
+
+import reactor.util.function.Tuple3;
+
+class GDUserProfileCommentRequest extends AbstractGDRequest<GDPaginator<GDComment>> {
```

Comment:
Rename to `GDUserCommentsRequest` (with an "s" to Comments)

Reply:
Changed to _GDUserCommentsRequest_